addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -13,6 +13,8 @@ class PaymentAcquirerStripe(models.Model):
 
     stripe_secret_key = fields.Char(required_if_provider='stripe')
     stripe_publishable_key = fields.Char(required_if_provider='stripe')
+    image_url = fields.Char("Image URL", required_if_provider='stripe',
```

Comment:
Why not use the company logo? Also I don't think we want to force the user to set an image.